### PR TITLE
convert .drone.yml to drone 1.0

### PR DESCRIPTION
**Pull request by individual-it**
## Description
double quoted lines 400 & 407 and ran `drone convert --save .drone.yml`

## Related Issue
part of #1731

## Motivation and Context
make CI pass

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4589/

<p>


































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4591/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4591/

<p>





















</p></details>

File: .drone.yml [hunk 1]
```diff
@@ -1,412 +1,2189 @@
...
+  - refs/pull/**
+  - refs/pull-requests/**
+  - refs/merge-requests/**
+  - refs/heads/master
+  - refs/heads/release*
+  - refs/heads/develop*
+
+---
+kind: pipeline
+name: matrix-14
```
Comment (DeepDiver1975) [inline .drone.yml]: rename missing - maybe publish-npm-and-demo-system

should only be executed on push

Comment (individual-it) [reply .drone.yml]: done

**Pull request merged**